www/docs/en/dev/plugin_ref/spec.md

```diff
@@ -17,700 +17,489 @@ license: >
+  arch(string) | ==windows== <br/> Allowed values: `x86`, `x64` or `ARM`. <br/> Indicates that the framework should only be included when building for the specified architecture. 
+  device-target(string) | ==windows== <br/> Allowed values: `win` (or `windows`), `phone` or `all`. <br/>  Indicates that the framework should only be included when building for the specified target device type.
+  versions(string) | ==windows== <br/> Indicates that the framework should only be included when building for versions that match the specified version string. Value can be any valid node semantic version range string.
+  target-dir(string) | ==windows== <br/>  Indicates a subdirectory into which the framework should be copied. In practice, this is most important when plugin contains different framework versions for different chip architectures or device targets, but which have the same name. This allows you to specify different subfolders for each framework version so that they don't overlap each other.
+
+  Examples:
+
+  For iOS: 
+  ```
+  <framework src="libsqlite3.dylib" />
+  <framework src="social.framework" weak="true" />
+  <framework src="relative/path/to/my.framework" custom="true" />
+  ```
+
+  On Android (as of cordova-android@4.0.0), framework tags are used to include Maven dependencies, or to include bundled library projects.
+  ```
+  <!-- Depend on latest version of GCM from play services -->
+  <framework src="com.google.android.gms:play-services-gcm:+" />
+  <!-- Depend on v21 of appcompat-v7 support library -->
+  <framework src="com.android.support:appcompat-v7:21+" />
+  <!-- Depend on library project included in plugin -->
+  <framework src="relative/path/FeedbackLib" custom="true" />
+  ```
+
+  Framework can also be used to have custom .gradle files sub-included into the main project's .gradle file:
+  ```
+  <framework src="relative/path/rules.gradle" custom="true" type="gradleReference" />
+  ```
+
+  On Windows, using `custom='true'` and `type='projectReference'` will add a reference to the project which will be added to the compile+link steps of the cordova project.  This essentially is the only way currently that a 'custom' framework can target multiple architectures as they are explicitly built as a dependency by the referencing cordova application.
+  ```
+  <framework src="path/to/project/LibProj.csproj" custom="true" type="projectReference"/>
+  ```
+
+  Examples of using these Windows specific attributes:
+  ```  
+  <framework src="src/windows/example.dll" arch="x64" />
+  <framework src="src/windows/example.dll" versions=">=8.0" />
+  <framework src="src/windows/example.vcxproj" type="projectReference" target="win" />
+  <framework src="src/windows/example.vcxproj" type="projectReference" target="all" versions="8.1" arch="x86" />
+  <framework src="src/windows/example.dll" target-dir="bin/x64" arch="x64" custom="true"/>
+  ```
 
-        <asset src="www/new-foo.js" target="js/experimental/foo.js" />
+### info
```

Comment:
I would mention that the contents of this tag will get printed out when the CLI installs the plugin